symbols/arm9.yml

```diff
@@ -6231,6 +6254,59 @@ arm9:
         Note: unverified, ported from Irdkwia's notes
         
         r0: dungeon ID
+    - name: CanLearnIqSkill
+      address:
+        EU: 0x2059054
+        NA: 0x2058CD8
+      description: |-
+        Returns whether an IQ skill can be learned with a given IQ amount or not.
+        
+        If the specified amount is 0, it always returns false.
+        
+        r0: IQ amount
+        r1: IQ skill
+        return: True if the specified skill can be learned with the specified IQ amount.
+    - name: GetLearnableIqSkills
+      address:
+        EU: 0x2059080
+        NA: 0x2058D04
+      description: |-
+        Determines the list of IQ skills that a given monster can learn given its IQ value.
+        
+        The list of skills is written in the array specified in r0. The array has 69 slots in total. Unused slots are set to 0.
+        
+        r0: (output) Array where the list of skills will be written
+        r1: Monster species
+        r2: Monster IQ
+        return: Amount of skills written to the output array
+    - name: DisableIqSkill
+      address:
+        EU: 0x2059120
+        NA: 0x2058DA4
+      description: |-
+        Disables an IQ skill.
+        
+        r0: Pointer to the bitarray containing the list of enabled IQ skills
+        r1: ID of the skill to disable
+    - name: EnableIqSkill
+      address:
+        EU: 0x2059170
+        NA: 0x2058DF4
+      description: |-
+        Enables an IQ skill and disables any other skills that are incompatible with it.
+        
+        r0: Pointer to the bitarray containing the list of enabled IQ skills
+        r1: ID of the skill to enable
+    - name: GetSpeciesIqSkill
+      address:
+        EU: 0x20591E4
+        NA: 0x2058E68
+      description: |-
+        Gets the <index>th skill on the list of IQ skills that a given monster species can learn.
```

Comment:
Is this zero-indexed? I assume so, but probably best to specify.

Reply:
:+1: